src/api/workspaces/queries.ts

```diff
@@ -113,6 +113,19 @@ export const MUTATION_LEAVE_WORKSPACE = `
   }
 `;
 
+
+// language=GraphQL
+/**
+ * Mutation to delete workspace
+ */
+export const MUTATION_DELETE_WORKSPACE = `
+ mutation leaveWorkspace(
+   $workspaceId: ID!
+ ) {
+   deleteWorkspace(workspaceId: $workspaceId)
```

Comment:
so `leaveWorkspace` or `deleteWorkspace`? Why they are different?

Reply:
any user can leave the workspace if he/she is not the last admin but only when there is a last admin who wants to leave the workspace the only way is to delete that workspace.
That why it's different.

Reply:
so this method is for leaving or deletion? It looks inconsistent:

```
/**
  * Mutation to delete workspace
  */
```

  
```
MUTATION_DELETE_WORKSPACE
```  

```
leaveWorkspace(
```

```
deleteWorkspace(
```